src/trello.coffee

```diff
@@ -1,112 +1,202 @@
+  ensureConfig msg.send
+  id = lists[list_name.toLowerCase()].id
+  msg.reply "I couldn't find a list named: #{list_name}." unless id
+  if id
+    trello.put "/1/cards/#{card_id}/idList", {value: id}, (err, data) ->
+      msg.reply "Sorry boss, I couldn't move that card after all." if err
+      msg.reply "Yep, ok, I moved that card to #{list_name}." unless err
+
+addDescription = (msg, card_id, desc) ->
+  ensureConfig msg.send
+  trello.put "/1/cards/#{card_id}/desc", {value: desc}, (err, data) ->
+      msg.reply "Sorry boss, I couldn't update that card after all." if err
+      msg.reply "Yep, ok, I updated that card for you." unless err
+
+addComment = (msg,card_id, comment, usr) ->
+  ensureConfig msg.send
+  trello.post "/1/cards/#{card_id}/actions/comments",{text: usr+" commented via Slack:"+comment}, (err, data) ->
+      msg.reply "Sorry, I was unable to do that." if err
+      msg.reply "Gladly! That comment has been added" unless err
+
+module.exports = (robot) ->
+  # fetch our board data when the script is loaded
+  ensureConfig console.log
+  trello.get "/1/boards/#{process.env.HUBOT_TRELLO_BOARD}", (err, data) ->
+    board = data
+    trello.get "/1/boards/#{process.env.HUBOT_TRELLO_BOARD}/lists", (err, data) ->
+      for list in data
+        lists[list.name.toLowerCase()] = list
+    trello.get "/1/boards/#{process.env.HUBOT_TRELLO_BOARD}/members", (err, data) ->
+      for member in data
+        members[member.fullName.toLowerCase().split " ", 1] = member
+
+
+  robot.respond /trello new [""'‘](.+)["”'’]\s(.*)/i, (msg) ->
+    ensureConfig msg.send
+    card_name = msg.match[2]
+    list_name = msg.match[1]
+
+    if card_name.length == 0
+      msg.reply "You must give the card a name"
+      return
+
+    if list_name.length == 0
+      msg.reply "You must give a list name"
+      return
+    return unless ensureConfig()
+
+    createCard msg, list_name, card_name
+
+  robot.respond /trello list ["“'‘](.+)["”'’]/i, (msg) ->
+    showCards msg, msg.match[1]
+
+  robot.respond /trello move (\w+) ["“'‘](.+)["”'’]/i, (msg) ->
+    moveCard msg, msg.match[1], msg.match[2]
+
+  robot.respond /trello comment (\w+) ["“'‘]((.+|\n)+)["”'’]/i, (msg) ->
+    addComment msg, msg.match[1], msg.match[2], msg.message.user.name
+
+  robot.respond /trello description (\w+) ["“'‘]((.+|\n)+)["”'’]/i, (msg) ->
+    addDescription msg, msg.match[1], msg.match[2]
+
+  robot.respond /trello get (\w+)['’]s cards/i, (msg) ->
+    getMemberCards msg, msg.match[1]
+    
+  robot.respond /trello search ["“'‘]((.+|\n)+)["”'’]/i, (msg) ->
+     search msg, msg.match[1]
+
+  robot.respond /trello list lists/i, (msg) ->
+    msg.reply "Here are all the lists on your board."
+    Object.keys(lists).forEach (key) ->
+      msg.send " * " + key
+
+  robot.respond /trello list members/i, (msg) ->
+    msg.reply "Here are all the members of the board"
+    Object.keys(members).forEach (member)->
+  msg.send " * " + member

```

Comment:
the indent here looks off, double check this. IIRC coffeescript is indentation based, like python and won't put this line in the function above.